token-service/services/ForecastApiService.js

```diff
@@ -0,0 +1,67 @@
+const { Curl } = require('node-libcurl');
+
+const ServiceForecastAPI = {
```

Comment:
why name of a file is ForecastApiService and obj name is ServiceForecastAPI?

Reply:
Done. This variable was renamed.